back-end/routes/api.php

```diff
@@ -19,4 +19,5 @@
 });
 
 Route::get('students', 'StudentController@get');
-Route::get('/product', "ProductController@get");
+Route::get('/product', 'ProductController@get');
+Route::post('/insert-product', 'ProductController@create');
```

Comment:
path 'insert-product' เปลี่ยนเป็น 'product' ธรรมดาดีกว่า Best Practice แนะนำมา https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/ ตรงหัวข้อ Use nouns instead of verbs in endpoint paths

Reply:
คือมันสามารถตั้งชื่อซ้ำกับ get ได้ถูกมะ แค่ใช้คำให้เป็น noun